client/js/lounge.js

```diff
@@ -844,7 +844,7 @@ $(function() {
 
 		socket.emit("input", {
 			target: chat.data("id"),
-			text: "/whois " + name
+			text: "/whois " + name + " " + name
```

Comment:
@astorije a comment in the code here, explaining why the name is repeated, might be helpful in the future